docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx

```diff
@@ -47,6 +48,66 @@ chain ID after deployment.**
 
 :::
 
+## Network RPCs
+
+<Tabs>
+  <TabItem value='iotaEVM' label='IOTA EVM'>
+    <AddToMetaMaskButton {...Networks['iota']}/>
+    <NetworkInfo.Evm {...Networks['iota']}/>
+  </TabItem>
+  <TabItem value='iotaEVMTestnet' label='IOTA EVM Testnet'>
+    <AddToMetaMaskButton {...Networks['iota_testnet']}/>
+    <NetworkInfo.Evm {...Networks['iota_testnet']}/>
+  </TabItem>
+  <TabItem value='shimmerEVM' label='ShimmerEVM'>
+    <AddToMetaMaskButton {...Networks['shimmer']}/>
+    <NetworkInfo.Evm {...Networks['shimmer']}/>
+  </TabItem>
+  <TabItem value='shimmerEVMTestnet' label='ShimmerEVM Testnet'>
+    <AddToMetaMaskButton {...Networks['shimmer_testnet']}/>
+    <NetworkInfo.Evm {...Networks['shimmer_testnet']}/>
+  </TabItem>
+</Tabs>
+
+## IOTA EVM Tools
+
+The following tools are **only available on IOTA EVM**.
+
+### EVM Toolkit
+
+You can use the [IOTA EVM Toolkit](https://evm-toolkit.evm.iotaledger.net) to withdraw assets from IOTA EVM to IOTA L1.
+It also includes a wrapper IOTA <-> wIOTA.
+
+### MultiSig Walllets
+
+If you require and additional level of security, you can use the [Safe{} Wallet](https://safe.iotaledger.net/) as a
+Multisig solution on IOTA EVM.
+
+### Oracles
+
+If your project requires [Oracles](/build/oracles/) to provide data from the outside world, you find both Pyth and Supra have integrated IOTA EVM.
+
+<OraclesContractData />
+
+### Multicall3
+
+If you need to aggregate results from multiple contract reads into a single JSON-RPC request or execute multiple
+state-changing calls in a single transaction, you can use the [Multicall3 contract](https://explorer.evm.iota.org/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract).
+
+### Subgraphs
```

Comment:
We have self hosted subgraphs on ShimmerEVM. Should we mention nakams fork here?

Reply:
I think the article is a better place to do so, wdyt?

Reply:
Yeah sounds good to me 👍